plugins/analysis-icu/src/internalClusterTest/java/org/elasticsearch/index/mapper/ICUCollationKeywordFieldMapperIT.java

```diff
@@ -306,12 +306,13 @@ public void testIgnoreWhitespace() throws Exception {
 
         assertAcked(client().admin().indices().prepareCreate(index).setMapping(builder));
 
-        indexRandom(
-            true,
-            client().prepareIndex(index).setId("1").setSource("{\"id\":\"1\",\"collate\":\"foo bar\"}", XContentType.JSON),
-            client().prepareIndex(index).setId("2").setSource("{\"id\":\"2\",\"collate\":\"foobar\"}", XContentType.JSON),
-            client().prepareIndex(index).setId("3").setSource("{\"id\":\"3\",\"collate\":\"foo-bar\"}", XContentType.JSON)
-        );
+        indexRandom(true, client().prepareIndex(index).setId("1").setSource("""
+            {"id":"1","collate":"foo bar"}
```

Comment:
I don't think this particular file is more readable for these changes. What do you think about keeping the old version?